server/utils/getEventDetailsViaId.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/models");
+
+// Returns a promise that resolves to an object containing the details of the
+// event. This function can be optimized by hooking it onto a cache object
```

Comment:
Good eye to the future, I think having a cache for event details makes a lot of sense